src/containers/auth/forgotPassword/index.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { PureComponent } from 'react';
+import Modal from 'react-modal';
+import { FaTimes } from 'react-icons/fa';
+import { IoIosCheckmarkCircle } from 'react-icons/io';
+import { CircleSpinner } from 'react-spinners-kit';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+import { forgotPasswordRequest } from '../../../services/auth';
+import validateAuth from '../../../validation/auth';
+import ShowMessage from '../showMessage/index';
+
+Modal.setAppElement('#root');
+
+class ForgotPassword extends PureComponent {
+  state = {
+    status: null,
+    data: null,
+    email: null,
+    message: null,
+    pending: false,
+  };
+
+  send = async email => {
+    const errors = validateAuth({ email });
+    this.setState({
+      pending: true,
+    });
+    if (!errors.email) {
+      const response = await forgotPasswordRequest({ email });
+      const { data, status } = response;
+      this.setState({
+        status,
+        data,
+      });
+    } else {
+      this.setState({
+        message: errors.email,
+      });
+    }
+    this.setState({
+      pending: false,
+    });
+  };
+
+  onChangeInput = e => {
+    const email = e.target.value;
+    this.setState({
+      email,
+    });
```

Comment:
You can simplify this code like:
```
 onChangeInput = e => {
    this.setState({
      email: e.target.value
    });
```
or make this code more flexible, like:
```
 onChangeInput = e => {
    this.setState({
      [e.target.name]: e.target.value
    });
```

Reply:
done